content/api/webhooks.md

```diff
@@ -504,170 +549,258 @@ Sent whenever a referral is converted.
+
 ```json
-{  
-    "id":"57731b5ee4b07320b5c0980a",
-    "type":"referral.converted",
-    "tenantAlias":"aohgcctyskc0p",
-    "live":true,
-    "created":1467161438453,
-    "data":{  
-        "id":"57731b43e4b07320b5c097ec",
-        "referredUser":"57731b42e4b08b14ab97a0c5",
-        "referrerUser":"5773073ee4b08b14ab979fb8",
-        "referralCodeUsed":"LORETTABURKE10" ,
-        "shareLinkUsed":"http://ssqt.co/mPbcF5",
-        "referredReward":null,
-        "referrerReward":null,
-        "moderationStatus":"PENDING",
-        "dateReferralStarted":1467161411028,
-        "dateReferralPaid":1467161438415,
-        "dateReferralEnded":null,
-        "dateModerated":1467161411027,
-        "referredModerationStatus":"PENDING",
-        "referrerModerationStatus":"PENDING",
-        "fraudSignals": {
-          "name": {
-            "message": "Referrer and referred have very similar names",
-            "score": 50
-          }
-        }
+{
+  "id": "57731b5ee4b07320b5c0980a",
+  "type": "referral.converted",
+  "tenantAlias": "aohgcctyskc0p",
+  "live": true,
+  "created": 1467161438453,
+  "data": {
+    "id": "57731b43e4b07320b5c097ec",
+    "referred": {
+      "id": "5773073ee4b08b14ab979fb8",
+      "accountId": "5773073ee4b08b14ab979fb8"
+    },
+    "referrer": {
+      "id": "577306eae4b08b14ab979f70",
+      "accountId": "577306eae4b08b14ab979f70"
+    },
+    "referralCodeUsed": "LORETTABURKE10",
+    "shareLinkUsed": "http://ssqt.co/mPbcF5",
+    "moderationStatus": "PENDING",
+    "dateReferralStarted": 1467161411028,
+    "dateConverted": 1467161438415
+  }
+}
+```
+
+#### <span class="label">Classic only</span> referral.converted if you are using the classic program
+
+```json
+{
+  "id": "57731b5ee4b07320b5c0980a",
+  "type": "referral.converted",
+  "tenantAlias": "aohgcctyskc0p",
+  "live": true,
+  "created": 1467161438453,
+  "data": {
+    "id": "57731b43e4b07320b5c097ec",
+    "referredUser": "57731b42e4b08b14ab97a0c5",
+    "referrerUser": "5773073ee4b08b14ab979fb8",
+    "referred": {
+      "id": "5773073ee4b08b14ab979fb8",
+      "accountId": "5773073ee4b08b14ab979fb8"
+    },
+    "referrer": {
+      "id": "577306eae4b08b14ab979f70",
+      "accountId": "577306eae4b08b14ab979f70"
+    },
+    "referralCodeUsed": "LORETTABURKE10",
+    "shareLinkUsed": "http://ssqt.co/mPbcF5",
+    "moderationStatus": "PENDING",
+    "dateReferralStarted": 1467161411028,
+    "dateReferralPaid": 1467161438415,
+    "dateConverted": 1467161438415,
+    "dateReferralEnded": null,
+    "dateModerated": 1467161411027,
+    "referredModerationStatus": "PENDING",
+    "referrerModerationStatus": "PENDING",
+    "fraudSignals": {
+      "name": {
+        "message": "Referrer and referred have very similar names",
+        "score": 50
+      }
     }
+  }
 }
 ```
 
-### referral.ended
+### <span class="label">Classic only</span> referral.ended
 
 Sent whenever a referred user, or referral, is cancelled.
```

Comment:
This doesn't actually make any sense without context. We can't "cancel" Referred Users--the way this is worded implies that a referral ended webhook is sent if a Referred User is deleted, which is not the case. I know this is an old Payment Provider-only use case, so please indicate that in the description as well.